docs/sources/flow/reference/components/otelcol.auth.sigv4.md

```diff
@@ -0,0 +1,171 @@
+---
+title: otelcol.auth.sigv4
+---
+
+# otelcol.auth.sigv4
+
+`otelcol.auth.sigv4` exposes a `handler` that can be used by other `otelcol`
+components to authenticate requests to AWS services using the AWS Signature Version 4 (SigV4) protocol. 
+See [here](https://docs.aws.amazon.com/general/latest/gr/signing-aws-api-requests.html) for more
+information about SigV4.
+
+> **NOTE**: `otelcol.auth.sigv4` is a wrapper over the upstream OpenTelemetry
+> Collector `sigv4auth` extension. Bug reports or feature requests will be
+> redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.auth.sigv4` components can be specified by giving them
+different labels.
+
+> **NOTE**: The Agent must have valid AWS credentials as used by the 
+[AWS SDK for Go](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials).
+
+## Usage
+
+If the exporter URL does not contain enough information about the region and service, 
+the minimum arguments are:
+```river
+otelcol.auth.sigv4 "LABEL" {
+  region  = "REGION"
+  service = "SERVICE"
+}
+```
+
+In some cases the exporter URL does contain enough information about the region and service as 
+described [here](#arguments). In that case there is no need to specify any arguments to this component:
+```river
+otelcol.auth.sigv4 "LABEL" {
+}
+```
```

Comment:
To follow the recommendations documented in #3145, this section should be composed of nothing but a River code block with required arguments. 

We can document when region/service may be required in the Arguments block without providing any information here.